lib/static/css/main.css

```diff
@@ -713,6 +713,46 @@ pre code {
   padding: 1.25rem 1.5rem;
 }
 
+/* "Copy" code block button */
+pre {
+  position: relative;
+}
+
+pre .btn-icon {
```

Comment:
Let's change all these to camelCase to be consistent with the current CSS.

Reply:
fixed, seemed like BEM `__` e.g. `btnIcon__body` was ok though?